src/serialization.jl

```diff
@@ -0,0 +1,34 @@
+struct Save
+    filename::String
+    method
+end
+
+Save(filename="machine.jlso"; method=serialize) =
+    Save(filename, method)
+
+
+IterationControl.@create_docs(Save,
+             header="Save(filename=\"machine.jlso\")",
+             example="Save(\"run3/machine.jlso\", compression=:gzip)",
+             body="Save the current state of the machine being iterated to "*
```

Comment:
I see the `header` and `example` are not consistent - one uses `filename` as kwarg, other as regular arg. 

Okay, I guess you've not looked at this doc-strong yet.